cmd/speep/main.go

```diff
@@ -73,6 +73,7 @@ func HandleGet(getCmd *flag.FlagSet, all *bool, keyword *string) error {
 		// Return full shortcut list
 		shortcuts, err := shortcuts.GetShortcuts()
 		if err != nil {
+			err = fmt.Errorf("Failed to acquire the existing list: %w", err)
 			return err
```

Comment:
Whenever you have an assignment before a return you can just return the stuff on the right hand side of the equals instead
```suggestion
			return fmt.Errorf("Failed to acquire the existing list: %w", err)
```

This comment applies throughout the PR.

Reply:
Thank you all addressed!